CHANGELOG.md

```diff
@@ -55,6 +55,7 @@ To learn more about active deprecations, we recommend checking [GitHub Discussio
 
 - **General:**: Add ScaledObject/ScaledJob names to output of `kubectl get triggerauthentication/clustertriggerauthentication` ([#796](https://github.com/kedacore/keda/issues/796))
 - **General:**: Add standalone CRD generation to release workflow ([#2726](https://github.com/kedacore/keda/issues/2726))
+- **Cloud Tasks Scaler**: Add functionality to scale based on the queue length ([#3613](https://github.com/kedacore/keda/issues/3613))
```

Comment:
As this is a new scaler, [based on the style guide](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#Changelog), it should be like:
```suggestion
- **General:** Introduce new Cloud Tasks Scaler ([#3613](https://github.com/kedacore/keda/issues/3613))
```

Reply:
I would use: `**General:** Introduce new Google Cloud Tasks Scaler` to make it clear what is this scaler about.